app/assets/stylesheets/custom.scss

```diff
@@ -134,6 +134,7 @@ h1, h4 {
   padding: $base-spacing-unit;
   text-align: center;
   position: fixed;
+	z-index: 1030;
```

Comment:
Strange intendation :woman_shrugging: If you change that  I can merge it later.

Reply:
Ok, I copied the line, so I get it now that I copied a tab. When I make tabs and spaces visible in my editor then I see other tabs. So I take this chance to replace all tabs with spaces. Do not be surprised ;)